.ci/hipify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright(C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+for DIR in ./src ./include
+do
+    for EXT in cpp h hpp cu
+    do
+        find ${DIR} -name "*.${EXT}" -exec sh -c '
+          echo "hipifying $1 in-place"
+           hipify-perl "$1" -inplace &
```

Comment:
Where do we get `hipify-perl` script?

Reply:
It is installed when installing HIP.
https://github.com/ROCm/HIP/blob/master/INSTALL.md

Reply:
Removed.